svc.py

```diff
@@ -0,0 +1,56 @@
+from sklearn import svm
+import sys
+import numpy as np
+
+feature_file = sys.argv[1]
```

Comment:
Python convention is to put this type of code in a ```main()``` method and call that method.